tests/torch/fx/test_ptq_params.py

```diff
@@ -0,0 +1,155 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import torch
+from torch import nn
+
+from nncf.common.graph.patterns import GraphPattern
+from nncf.common.graph.patterns.manager import PatternsManager
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.graph.transformations.commands import TransformationType
+from nncf.common.utils.backend import BackendType
+from nncf.experimental.torch.fx.commands import FXApplyTransformationCommand
+from nncf.experimental.torch.fx.nncf_graph_builder import GraphConverter
+from nncf.parameters import TargetDevice
+from nncf.quantization.algorithms.min_max.algorithm import MinMaxQuantization
+from nncf.quantization.algorithms.min_max.torch_fx_backend import FXMinMaxAlgoBackend
+from nncf.scopes import IgnoredScope
+from nncf.torch.graph.graph import PTNNCFGraph
+from nncf.torch.graph.graph import PTTargetPoint
+from nncf.torch.graph.operator_metatypes import PTCatMetatype
+from nncf.torch.graph.operator_metatypes import PTConv2dMetatype
+from nncf.torch.graph.operator_metatypes import PTLinearMetatype
+from nncf.torch.graph.operator_metatypes import PTSoftmaxMetatype
+from tests.common.quantization.metatypes import CatTestMetatype
+from tests.common.quantization.metatypes import Conv2dTestMetatype
+from tests.common.quantization.metatypes import LinearTestMetatype
+from tests.common.quantization.metatypes import SoftmaxTestMetatype
+from tests.cross_fw.test_templates.test_ptq_params import TemplateTestPTQParams
+from tests.torch.fx.helpers import get_single_conv_nncf_graph
+from tests.torch.fx.helpers import get_torch_fx_model
+from tests.torch.helpers import create_bn
+from tests.torch.helpers import create_conv
+from tests.torch.ptq.helpers import get_single_no_weight_matmul_nncf_graph
+
+
+def get_hw_patterns(device: TargetDevice = TargetDevice.ANY) -> GraphPattern:
+    return PatternsManager.get_full_hw_pattern_graph(backend=BackendType.TORCH_FX, device=device)
+
+
+def get_ignored_patterns(device: TargetDevice = TargetDevice.ANY) -> GraphPattern:
+    return PatternsManager.get_full_ignored_pattern_graph(backend=BackendType.TORCH_FX, device=device)
+
+
+class LinearTestModel(nn.Module):
+    INPUT_SIZE = None
+
+    def __init__(self):
+        super().__init__()
+        self.conv1 = create_conv(3, 3, 1)
+        self.bn1 = create_bn(3)
+        self.relu = nn.ReLU()
+        self.avg_pool = nn.AdaptiveAvgPool2d(1)
+        self.conv2 = create_conv(3, 1, 1)
+        self.bn2 = create_bn(1)
+
+    def forward(self, x):
+        x = self.relu(self.conv1(x))
+        x = self.bn1(x)
+        x = self.avg_pool(x)
+        x = self.relu(self.conv2(x))
+        x = self.bn2(x)
+        return x
```

Comment:
@rk119, could you please put this model to `nncf/tests/torch/test_models/synthetic.py` with a name LinearPTQParamsTestModel and reuse it in both Torch and TorchFX tests?

Reply:
> @rk119, could you please put this model to `nncf/tests/torch/test_models/synthetic.py` with a name LinearPTQParamsTestModel and reuse it in both Torch and TorchFX tests?

Alright.